bundle.sh

```diff
@@ -17,7 +17,7 @@ for chart in charts/*; do
     echo "File $JSON_SCHEMA does not exist"
     touch "$JSON_SCHEMA"
   fi
-  yq eval-all --tojson "select(fileIndex == 0) * select(filename == \"$YAML_SCHEMA\")" definitions.schema.yaml "${chart}"/values.schema.yaml > "$JSON_SCHEMA"
+  yq eval-all -o=json "select(fileIndex == 0) * select(filename == \"$YAML_SCHEMA\")" definitions.schema.yaml "${chart}"/values.schema.yaml > "$JSON_SCHEMA"
```

Comment:
I don't see -o as a valid flag for yq eval-all. All I can find is the shorthand of `--tojson` as `-j`.

https://mikefarah.gitbook.io/yq/commands/evaluate-all

Otherwise, seems fine
@jgomez-jb

Reply:
@landon-taylor thanks for the review. Yeah, I agree that per the documentation it does not need to change.
But the snap `yq` 4.16.1 is throwing this message:
```
Flag --tojson has been deprecated, please use -o=json instead
```
I replaced it and tested it locally, and the warning no longer appeared.